coreapi/main_api.py

```diff
@@ -146,7 +147,8 @@ def FaceRecogniseInImage(request, filename):
                         id_name = identify_face(embedding=embedding, embedding_dict=embedding_dict)
                         facial_expression = FaceExp(img)
                         bounding_box = {"top": bb[1], "bottom": bb[3], "left": bb[0], "right": bb[2]}
-                        all_face_dict[id_name] = {"Bounding Boxes": bounding_box, "Facial Expression": facial_expression}
+                        all_face_dict[id_name+'_'+str(i)] = {"Bounding Boxes": bounding_box, "Facial Expression": facial_expression}
```

Comment:
can we think in different way to handle this without changing `id_name`?
RekoUI depends upon the data format being saved. so changes must be made to that repo accordingly.